lib/takeaway.rb

```diff
@@ -0,0 +1,25 @@
+class Takeaway
+    attr_reader :order
+    attr_reader :price
+
+    def initialize(dishes = Dishes.new)
```

Comment:
Nice use of dependency injection! This could be further utilised by using a mock when testing the takeaway class